app/uk/gov/hmrc/hmrcemailrenderer/templates/digitalcontact/digitalOptInConfirmation_NinoPTA.scala.html

```diff
@@ -41,10 +36,7 @@ <h2 style="margin: 0 0 30px; font-size: 19px;">You will never miss a tax letter<
 <h2 style="margin: 0 0 30px; font-size: 19px;">To read your tax letters</h2>
 
 <p style="margin: 0 0 30px; font-size: 19px;">You need to:</p>
-<ul>
-<li>sign in to HMRC services and select 'Messages'</li>
-<li>you can also use HMRC's free mobile app. Search for 'HMRC app' to download it</li>
-</ul>
+<p style="margin: 0 0 30px; font-size: 19px;">* sign in to HMRC services and select 'Messages'<br/>* you can also use HMRC's free mobile app. Search for 'HMRC app' to download it</p>
```

Comment:
back ticks?

Reply:
wasn't part of the requirements. I copy/paste from the ticket. If we need to change styling I think we have to create a separate ticket to address that.

Reply:
The copy pasting doesn't work I'm afraid :) back ticks copied from the Notes app in Mac OSX for instance are converted to straight ticks.